src/style.css

```diff
@@ -1 +1,84 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. You can just use padding for your links
```suggestion
```